web/src/components/widgets/Sidebar.tsx

```diff
@@ -0,0 +1,43 @@
+import { reposStore } from '@/store/repoStore'
+import { NavigationMenu } from '@kobalte/core/navigation-menu'
```

Comment:
a bit about structure this api.
kobalte is completely naked, so similar to shadcn (actually a port of shadcn for solid) we take those primitives from there:
https://www.solid-ui.com/docs/components/sidebar
to take a component from there I look for it in the repo:
https://github.com/stefan-karger/solid-ui/blob/main/apps/docs/src/registry/ui/sidebar.tsx
create a file sidebar.tsx in ui folder, paste, fix a path to clsx or the others libs and commit, so a prestyled component is ready.